### PR TITLE
[BFD-887] Update local dev to use postgres 12

### PR DESCRIPTION
**Pull request by jzulim**
### Change Details

This change switches local development docker instructions to utilize postgres 12. 
### Acceptance Validation

New docker container (Postgres 12) works without issue (i.e. Passing IT's) 

### External References

- [BFD-887](https://jira.cms.gov/browse/BFD-887)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Review by mackec1 — APPROVED** [review]
ship it!

File: README.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ git clone git@github.com:CMSgov/beneficiary-fhir-data.git ~/workspaces/bfd/benef
 </toolchains>
 ```
 4. Change to the `apps/` directory and `mvn clean install -DskipITs`. The flag to skip the integration tests is important here. You will need to have AWS access for the integration tests to work correctly.
-5. Set up a Postgres 11 database. The easiest way to set up a local database is with the following command. Data will be persisted between starts and stops in the `bfd_pgdata` volume.
+5. Set up a Postgres 12 database. The easiest way to set up a local database is with the following command. Data will be persisted between starts and stops in the `bfd_pgdata` volume.
```
Comment (JeremyIR) [inline README.md:90]: I coincidentally made this change in my PR for the postgres docker container, but I also added more in this area to explain running the automated script. LGTM. Ill resolve the merge conflict.

**Review by JeremyIR — APPROVED** [review]

**Review by JeremyIR — APPROVED** [review]
LGTM

**Review by keithdadkins — APPROVED** [review]
Assuming we will get our prod to 12 soon, this LGTM

**Review by mackec1 — COMMENTED** [review]
Ship it!

**Pull request merged**